### PR TITLE
fix percent-encoding of URI examples

### LINKED ISSUE #319: URI percent-encoding in examples
We have several examples that contain a client_id with the new client_id_scheme prefixing in URIs like this:

```
GET /authorize?
  response_type=vp_token
  &client_id=redirect_uri:https%3A%2F%2Fclient.example.org%2Fcb
  &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
  &presentation_definition=...
  &transaction_data=...
  &nonce=n-0S6_WzA2Mj HTTP/1.1
```

Shouldn't the client_id line be this?
`&client_id=redirect_uri%3Ahttps%3A%2F%2Fclient.example.org%2Fcb`

Similarly, the example in 5.10.4 contains `:` inside the client_metadata
```
 &client_metadata=%7B%22vp_formats%22:%7B%22jwt_vp_json%22:%
  7B%22alg%22:%5B%22EdDSA%22,%22ES256K%22%5D%7D,%22ldp
  _vp%22:%7B%22proof_type%22:%5B%22Ed25519Signature201
  8%22%5D%7D%7D%7D
```
and should be this unless I am mistaken?
```
&client_metadata=%7B%22vp_formats%22%3A%7B%22jwt_vp_json%2
2%3A%7B%22alg%22%3A%5B%22EdDSA%22%2C%22ES256K%22%5
D%7D%2C%22ldp_vp%22%3A%7B%22proof_type%22%3A%5B%22Ed
25519Signature2018%22%5D%7D%7D%7D
```

### PR DESCRIPTION
**Pull request by c2bo**
fixes #319
Fixes some minor inconsistencies in the examples that use percent-encoding in URIs.

### CODE REVIEW

File: openid-4-verifiable-presentations-1_0.md [hunk 1]
```diff
@@ -517,22 +517,22 @@ Confusing Verifiers using a Client Identifier Scheme with those using none can l
...
 
-* `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Authorization Request MUST NOT be signed. The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). Example Client Identifier: `redirect_uri:https%3A%2F%2Fclient.example.org%2Fcb`.
+* `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Authorization Request MUST NOT be signed. The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). Example Client Identifier: `redirect_uri:https://client.example.org/cb`.
```
Comment (peppelinux) [inline openid-4-verifiable-presentations-1_0.md:520]:
```suggestion
* `redirect_uri`: This value indicates that the Client Identifier (without the prefix `redirect_uri:`) is the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used). The Authorization Request MUST NOT be signed. The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#new_parameters). For instance, the Example Client Identifier `redirect_uri:https://client.example.org/cb` MUST be formatted as `redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb` using percent-encoding for URL parameters, as defined in [@!RFC6749].
```

**Review by peppelinux — APPROVED** [review]
just added a suggestion to further clarifying the format to use when using the client id url paramenter

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (c2bo) [reply openid-4-verifiable-presentations-1_0.md:520]: Yes, that would be my understanding as well and why I converted to a normal URI without percent encoding.

Comment (Sakurann) [reply openid-4-verifiable-presentations-1_0.md:520]: can you please accept my suggestion, so that I can approve it?

**Review by Sakurann — APPROVED** [review]

File: openid-4-verifiable-presentations-1_0.md (hunk 1 above)
Comment (bc-pi) [reply openid-4-verifiable-presentations-1_0.md]: what @Sakurann said BTW

**Pull request merged**